packages/core/strapi/lib/commands/build.js

```diff
@@ -15,7 +15,7 @@ module.exports = async ({ clean, optimization }) => {
   const dir = process.cwd();
   const config = loadConfiguration(dir);
 
-  const { serverUrl, adminPath } = getConfigUrls(config.get('server'), true);
+  const { serverUrl, adminPath } = getConfigUrls(config.server, true);
```

Comment:
shouldn't this be config.get ?

Reply:
No because it comes from `loadConfiguration(dir)` and not `createConfigProvider`.
In Strapi.js we have:
```js
    const appConfig = loadConfiguration(this.dir, opts);
    this.container.register('config', createConfigProvider(appConfig));
```